### ISSUE TITLE
Zombie Master kill count is unreadable

### ISSUE THREAD

**Issue by benjaminmkos** [CleverRaven/Cataclysm-DDA#27038]
**Describe the bug**  
There is a particular monster on my current character's kill count screen that is completely unreadable because it uses bright red font on a bright red background. I am ~reasonably sure that monster is "Zombie Master"
Screenshot attached below.

**To Reproduce**  
1. Kill a Zombie Master
2. Open the Kill Count screen with ')'

**Expected behavior**  
The Kill Count screen should have be readable entries for each monster.
I would think that either the background should be changed to black (see zombie brute / kreck entries in screenshot) or the font should be changed to black (see zombie burner entry in screenshot.)

**Screenshots**  
![killcountcoloring](https://user-images.githubusercontent.com/6393606/49712304-ce43e600-fc11-11e8-997c-d4e74e14db91.png)

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version 0.C-35090-g9d693be
 - Graphics version Tiles
 - Mods loaded: boats, extended realistic guns, vehicle additions pack

**Additional context**

**Comment by Inglonias** [comment]
I can actually read the screenshot if I lean in and squint. You're right. It's a zombie master. But I second this issue. It's not fun to read.

**Comment by Pupsi-Mupsi** [comment]
Thanks a lot for your confirmation.
> 
> I suggest changing the display in that window to show the symbol (in the color associated with the monster type) and print the name in a standard color (e.g. white on black).

I like your idea!
Like this?

```
for( const auto &entry : kill_counts ) {
        std::ostringstream buffer;
        buffer << "<color_" << std::get<2>( entry.first ) << ">";
        buffer << std::get<1>( entry.first ) << "</color>" << " " << std::get<0>( entry.first );
```
